build.gradle

```diff
@@ -1072,7 +1072,6 @@ project(':core') {
     implementation libs.argparse4j
     implementation libs.commonsValidator
     implementation libs.jacksonDatabind
-    implementation libs.jacksonModuleScala
```

Comment:
for future reader, the dependency was introduced by https://github.com/apache/kafka/commit/47a9871ef65a13f9d58d5ea216de340f7e123da5, and https://github.com/apache/kafka/commit/b6183a41342c765daec9c88f9d2723a221131960 rewrite the tool by java so we don't need to use `jackson-module-scala` anymore.

Reply:
for another, we will file a PR to fix the license check. The PR #18004 also introduces a license warning.

Reply:
#18950